sass/components/_slider.scss

```diff
@@ -53,7 +62,9 @@
         width: 70%;
         opacity: 0;
 
-        p { color: $slider-bg-color-light; }
+        p {
+          color: rgba(255, 255, 255, 0.75);
+        }
```

Comment:
Where is the class `.slides` set? From noUISlider? I did not find any usage for this class.

Reply:
Yes, it's from noUISlider